spec/model/base_spec.cr

```diff
@@ -685,6 +685,19 @@ describe Jennifer::Model::Base do
     end
   end
 
+  describe "::upsert" do
+    it "do nothing on conflict when inserting" do
+      contact = Factory.build_contact
+      contact.description = "unique"
+      contact.save.should be_true
+
+      c = Factory.build_contact(age: 12, description: "unique")
+      Contact.upsert([c])
```

Comment:
let's upsert 2 records and check that we inserted only one that doesn't violate uniqueness constraint

Reply:
Yeah I can add this!